src/components/SamlConfiguration/index.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Index from './index';
+import LmsApiService from './../../data/services/LmsApiService';
+import { REQUIRED_DATA_FIELDS } from './SamlProviderDataForm';
+
+const formData = new FormData();
+REQUIRED_DATA_FIELDS.forEach(field => formData.append(field, 'testdata'));
+
+describe('<SamlConfiguration /> ', () => {
+  it('creating provider config states', () => {
+    const wrapper = shallow(<Index enterpriseId="testEnterpriseId" />);
```

Comment:
I'm confused by using Index here instead of `<SamlConfiguration`, Not really sure how a base Index component affects tests honestly.

Reply:
Ahh, no good catch. That should probably be SamlConfiguration.